duckduckgo/iOS
# use correct icons and update privacy practices UI

Reviewer: Mia, Caine or Craig
Asana: https://app.asana.com/0/414235014887631/487711692211727
CC:

**Description**:

Apply the icons as per the spec in the Asana ticket and update the UI to better reflect what was in Zeplin - in particular removing the alarming "No terms of service found" which was displayed if no privacy practices were or if no TOS object was found.

**Steps to test this PR**:

Test 1 
1. Visit DuckDuckGo and ensure all icons are green on the overview and detail screens
1. Turn off Privacy Protection and check again

Test 2 
1. Visit http://private.brindy.org.uk - unencrypted, unknown privacy practices, no trackers
1. Visit falkirkrpg.org.uk/noxwall/mixed.html - mixed encryption, no trackers, unknown privacy practices (grey icon with cross)

Test 3
1. Visit https://cnn.com - redirects to https://edition.cnn.com 
1 Encrypted connection, networks and major networks blocked (green icons), unknown privacy practices
1. Turn off privacy protection
1. Encrypted connection, networks and major networks found (red icons), unknown privacy practices

Test 4
1. Visit soundcloud.com 
1. Note mixed privacy practices (grey icon with check)
